pkg/controller/proxyconfig/validation.go

```diff
@@ -25,7 +33,7 @@ func (r *ReconcileProxyConfig) ValidateProxyConfig(proxyConfig *configv1.ProxySp
 			return fmt.Errorf("invalid httpProxy URI: %v", err)
 		}
 		if scheme != proxyHTTPScheme {
-			return fmt.Errorf("httpProxy requires an %q URI scheme", proxyHTTPScheme)
+			return fmt.Errorf("httpProxy requires an '%s' URI scheme", proxyHTTPScheme)
```

Comment:
just leaning in here on a brief skim, but why change `%q` to the `'%s'`.  Seems like more characters for the same effect?

Reply:
@wking consistency with formatting of other error messages. I can go back and update the other errors to use `%q`, but I'd prefer addressing that in a follow-on PR due to time constraints.

Reply:
I have no stake in this; don't hold things up on my account